experimenter/tests/integration/legacy/test_filter_expression_validation.py

```diff
@@ -1,14 +1,29 @@
-def test_filter_expressions_with_matching_firefox_versions(base_url, selenium):
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def filter_expression_path():
+    here = Path(__file__).cwd()
+    return Path(next(iter(here.glob("**/legacy/utils/filter_expression.js"))))
+
+
+def test_filter_expressions_with_matching_firefox_versions(
+    base_url, selenium, filter_expression_path
+):
     selenium.get("about:blank")
-    with open("legacy/utils/filter_expression.js") as js:
+    with Path(filter_expression_path).open() as js:
```

Comment:
`filter_expression_path` is already a `Path`, so you can call `.open()` directly.